### PR TITLE
Transactions overview page updated

### PR DESCRIPTION
**Pull request by amarantha-k**
Other than updates from open comments being discussed in the shared doc, this content is ready to be staged.

### CODE REVIEW

File: modules/learn/pages/data/distributed-transactions.adoc [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+:tabs:
+
+[abstract]
+{description}
+
+== Overview
+
+Couchbase transactions support ACID properties for protected actions on the database.
+
+*Atomicity* ensures that a transaction provides all-or-nothing semantics i.e.  either all the documents modified in a transaction are committed, or none of the changes are committed. If there is a failure during transaction execution (such as the client crashing) all its changes are rolled back.
```
Comment (RichardSmedley) [inline modules/learn/pages/data/distributed-transactions.adoc]: I'd put an em dash before the i.e., but a semi-colon or even a comma would be fine.
Just a nitpick, certainly not a blocker.

File: modules/learn/pages/data/distributed-transactions.adoc [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+For more information on using Query statements in transactions, see xref:n1ql:n1ql-language-reference/transactions.adoc[].
+
+NOTE: Take a look at the https://transactions.couchbase.com[Query Transaction Simulator] which demonstrates how query statements work in transactions.
+
+== Structure of a Transaction
+
+Every transaction has a beginning and a commit or a rollback at the end. 
+
+Every transaction consists of one or more KV operations and one or more query statements.
```
Comment (RichardSmedley) [inline modules/learn/pages/data/distributed-transactions.adoc]: Is that 'and' correct? A transaction could have zero Query statements?

Comment (amarantha-k) [reply modules/learn/pages/data/distributed-transactions.adoc]: How about: Every transaction consists of one or more KV operations, and optionally one or more query statements.

File: modules/learn/pages/data/distributed-transactions.adoc [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+Assuming that the 2 documents involved in this transaction live in two different nodes, here are the high-level steps that the transaction follows:
+
+image::transaction-mechanics-steps.png["Transaction mechanics explaining the high-level steps that a transaction follows"]
+
+Each execution of the transaction logic in an application is called an 'attempt' inside the overall transaction.
+
+=== Active Transaction Record Entries
+
+The first mechanic is that each of these attempts adds an entry to a metadata document in the Couchbase cluster. These metadata documents are called _Active Transaction Records_, or ATRs. 
+ATRs are created and maintained automatically and are easily distinguishable by their prefix `_txn:atr-`. They are viewable and should not be _modified externally_.
```
Comment (RichardSmedley) [inline modules/learn/pages/data/distributed-transactions.adoc]: I think `_txn:atr-` will need putting in the headers with a {} key here, to stop the underscore being parsed as an asciidoc italic (unless Antora is smarter than GH's asciidoc renderer?).
Speaking of italics, should the 'should not' part of the statement be added into the italics, so covering 5 words in place of just the last two?

File: modules/learn/pages/data/distributed-transactions.adoc [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+
+By default, the metadata documents are created in the default collection of the bucket of the first mutated document in the transaction. However, you can choose to use a named collection to store metadata documents. See <<Custom metadata collections>> for details.
+
+=== Staged Mutations
+
+The second mechanic is that mutating a document inside a transaction, does not directly change the body of the document. Instead, the post-transaction version of the document is staged alongside the document (technically in its xref:java-sdk:concept-docs/xattr.adoc[extended attributes] (XATTRs)). In this way, all changes are invisible to all parts of the Couchbase cluster until the commit point is reached.
+
+These staged document changes effectively act as a lock against other transactions trying to modify the document, preventing write-write conflicts.
+
+In Steps 2 and 3 in the illustration above, the transaction id and the content for the first and second mutations are staged in the xAttrs of their respective documents.
```
Comment (RichardSmedley) [inline modules/learn/pages/data/distributed-transactions.adoc]: caps for XATTRs

File: modules/learn/pages/data/distributed-transactions.adoc [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+
+|atrcollection
+|Specifies the collection where the active transaction record (ATR) is stored. The collection must be present. If not specified, the ATR is stored in the default collection in the default scope in the bucket containing the first mutated document within the transaction. See 
+xref:settings:query-settings.adoc#atrcollection_req[atrcollection] for details.
+|===
+
+== Related Topics
+
+* xref:java-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc[]
+* xref:cxx-txns:distributed-acid-transactions-from-the-sdk.adoc
```
Comment (RichardSmedley) [inline modules/learn/pages/data/distributed-transactions.adoc]: Two :: for ROOT module:
cxx-txns::distributed-

File: modules/learn/pages/data/distributed-transactions.adoc [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+* xref:dotnet-sdk:howtos:distributed-acid-transactions-from-the-sdk.adoc
```
Comment (RichardSmedley) [inline modules/learn/pages/data/distributed-transactions.adoc:315]: Do these xrefs need a []
added?
I don't tend to use them without link text, so I'm unsure.

Comment (amarantha-k) [reply modules/learn/pages/data/distributed-transactions.adoc:315]: If we don't specify the label, it uses the title of the linked page.

**Review by RichardSmedley — APPROVED** [review]
A couple of comments added on possible issues for Antora, but given time constraints no other nitpicks ;)
LGTM, nice document - logical and understandable look at txns.  :+1:

File: modules/learn/pages/data/distributed-transactions.adoc (hunk 6 above)
Comment (RichardSmedley) [reply modules/learn/pages/data/distributed-transactions.adoc:315]: Yes, gathered that - was more the lack then of an _empty_ [], which was used for the first of these bullets.
Is line 313 correct, or the others without []  ?

**Comment by amarantha-k** [comment]
Thank you, Richard! 
Thanks to Shivani, Graham, and Kamini for their time and input over the past week to get this document to the current shape. A few more updates are expected, but in the interest of time, I'll merge this PR and plan to create a new PR for further updates.

**Pull request merged**